test/lib/y2firewall/importer_strategies/suse_firewall.rb

```diff
@@ -73,14 +73,18 @@
         expect(dmz.interfaces).to eq(["eth2"])
       end
 
+      it "sets the default zone as the one with the 'any' interface" do
+        expect(firewalld.default_zone).to eql("dmz")
+      end
+
       context "and masquerade is disabled" do
         let(:masquerade) { "no" }
 
         it "configures the EXT zone as the public" do
           public_zone = firewalld.find_zone("public")
 
           expect(public_zone.interfaces).to eq(["eth0"])
-          expect(public_zone.services).to eq(["dhcp", "ssh", "samba", "vnc-server"])
```

Comment:
Any reason to not check services?

Reply:
I removed it because I checked the behavior in the next test, so not a strong one.